src/fancyline/tty/vt100.cr

```diff
@@ -1,5 +1,19 @@
 class Fancyline
   class Tty
+    lib LibC
+      struct Winsize
+        ws_row : UInt16    #  rows, in characters
+        ws_col : UInt16    #  columns, in characters
+        ws_xpixel : UInt16 #  horizontal size, pixels
+        ws_ypixel : UInt16 #  vertical size, pixels
+      end
+
+      IOC_OUT      = 0x40000000
+      IOCPARM_MASK =     0x1fff
+      TIOCGWINSZ   = IOC_OUT | ((sizeof(Winsize) & IOCPARM_MASK) << 16) | (('t'.ord) << 8) | 104
+      fun ioctl(fd : Int32, cmd : UInt64, winsize : Winsize*) : Int32
```

Comment:
`ioctl(3p)` says the prototype is actually (translated) `fun ioctl(fd : ::LibC::Int, request : ::LibC::Int, ...) : ::LibC::Int`. Not so much about the `Int32` stuff, but please change to use the `...` version to make it more robust.  This will probably break your `out winsize` below.